File: AirportManagement.API/AirportManagement.API/Models/Passangers.cs
using System;
namespace AirportManagement.API.Models
{
	public class Passangers
	{
		public int Id { get; set; }

		public string FirstName { get; set; }

		public string LastName { get; set; }

		public DateTime DateOfBirth { get; set; }

		public string CountryOfCitizenship { get; set; }

		public string CountryOfResidence { get; set; }

		public string PassportNumber { get; set; }

		public DateTime CreatedAt { get; set; }

		public DateTime UpdatedAt { get; set; }
	}
}

File: AirportManagement.SPA/src/app/_models/Passangers.ts
import { Booking } from "./Booking";
import { NoFlyList } from "./NoFlyList";
import { SecurityCheck } from "./SecurityCheck";

export interface Passangers {
    id: number;
    firstName: string;
    lastName: string;
    DateOfBirth: Date;
    CountryOfCitizenship: string;
    CountryOfResidence: string;
    passportNumber: string;
    CreatedAt: Date;
    UpdatedAt: Date;
    SecurityChecks: SecurityCheck[];
    Bookings: Booking[];
    NoFlyLists: NoFlyList[];
}
